### PR TITLE
#8 issue fixed temporary

### LINKED ISSUE #8: "Read our Code of Conduct" link opens a blank page
## 🐛 Bug Description

On the homepage (https://community.tailed.ca/), clicking the 'Read our Code of Conduct' link navigates to (https://community.tailed.ca/docs/CODE_OF_CONDUCT.md).

## 🔄 Steps to Reproduce

1. Go to https://community.tailed.ca/ and locate the Code of Conduct card.
2. Click on “Read our Code of Conduct” link.
3. Observe that the page is blank instead of showing the document.

## ✅ Expected Behavior

The code of conduct markdown file should display (either on the site or by redirecting to the GitHub page).

## ❌ Actual Behavior

The blank page is appearing on the screen.

## 📷 Screenshot 

Image

## 🖥️ Environment

**Laptop:**

- OS: [e.g. Windows 11]
- Browser: [e.g. Chrome, Firefox]

### Authentication State

- [ ] Logged in
- [ ] Logged out

### Feature Area

- [ ] Dashboard

---

### For Maintainers

**Priority:**

- [ ] Critical (Blocks core functionality)
- [ ] High (Affects many users)
- [ ] Medium (Affects some users)
- [ ] Low (Minor issue)

**Complexity:**

- [ ] Simple (< 1 day)
- [ ] Medium (1-3 days)
- [ ] Complex (> 3 days)

**Area:**

- [ ] Frontend
- [ ] Backend
- [ ] Infrastructure
- [ ] Documentation

### PR DESCRIPTION
**Pull request by VedPatel2811**
## 📋 Description

Update the "Read our Code of Conduct" and "Read CONTRIBUTING.md" link on the homepage to point directly to the GitHub .md files. This ensures users can access the Code of Conduct immediately, as the previous link to /docs/CODE_OF_CONDUCT.md was showing a blank page.

This is a temporary fix until the maintainers decide whether the Markdown should render on the site or continue redirecting to GitHub.

Fixes #8 

## 🔄 Type of Change

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 Documentation update
- [ ] 🎨 Style/UI changes
- [ ] ♻️ Code refactoring (no functional changes)
- [ ] ⚡ Performance improvements
- [ ] 🧪 Test updates
- [ ] 🔧 Build/CI changes

## 🧪 Testing

- [ ] Unit tests pass
- [ ] Integration tests pass
- [ ] Manual testing completed
- [ ] Cross-browser testing (if applicable)
- [ ] Mobile testing (if applicable)

### Test Details

Describe the tests you ran and how to reproduce them:

```bash
# Example test commands
npm test
npm run e2e
```

## 📷 Screenshots (if applicable)

| Before         | After         |
| -------------- | ------------- |
| before | after |

### Code Quality

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### Documentation

- [ ] I have made corresponding changes to the documentation
- [ ] I have updated the README if needed
- [ ] I have added/updated JSDoc comments for new functions

### Dependencies

- [ ] I have checked that no new dependencies were added unnecessarily
- [ ] All new dependencies are documented and justified

### Security

- [ ] I have considered security implications of my changes
- [ ] No sensitive information is exposed in the code
- [ ] API endpoints are properly secured (if applicable)

### Performance

- [ ] I have considered the performance impact of my changes
- [ ] No unnecessary re-renders or computations introduced
- [ ] Database queries are optimized (if applicable)

### Accessibility

- [ ] UI changes are accessible (proper ARIA labels, keyboard navigation, etc.)
- [ ] Color contrast meets accessibility standards
- [ ] Screen reader compatibility verified

## 🔗 Related Issues

- Closes #
- Related to #
- Depends on #

## 🚀 Deployment Notes

Any special deployment considerations:

- [ ] Requires database migration
- [ ] Requires environment variable changes
- [ ] Requires cache clearing
- [ ] Requires feature flag updates
- [ ] No special deployment steps needed

## 📝 Additional Notes

Any additional information that reviewers should know:

- Breaking changes and migration guide
- Known limitations or trade-offs
- Future improvements planned
- Dependencies on other PRs

## 🔍 Review Focus Areas

Please pay special attention to:

- [ ] Logic in `component/function X`
- [ ] Error handling
- [ ] Edge cases
- [ ] Performance implications
- [ ] Security considerations

---

### For Reviewers

**Review Checklist:**

- [ ] Code follows project conventions
- [ ] Tests are comprehensive
- [ ] Documentation is updated
- [ ] No security vulnerabilities introduced
- [ ] Performance impact is acceptable
- [ ] Accessibility standards met

### CODE REVIEW

File: src/pages/landing/page.tsx [hunk 1]
```diff
@@ -236,7 +236,7 @@ git push origin feat/amazing-contribution
                   <div className="mt-4">
                     <a
                       className="text-primary underline underline-offset-4"
-                      href="/CONTRIBUTING.md"
+                      href="https://github.com/VedPatel2811/tailed-community-dashboard/blob/main/CONTRIBUTING.md"
```
Comment (Franck-Junior) [inline src/pages/landing/page.tsx]: (fix) link is pointing to your fork

**Review by Franck-Junior — CHANGES_REQUESTED** [review]
